web/beacon-app/src/features/topics/api/topicsApiService.ts

```diff
@@ -2,15 +2,14 @@ import type { ApiAdapters } from '@/application/api/ApiAdapters';
 import type { Request } from '@/application/api/ApiService';
 import { getValidApiError, getValidApiResponse } from '@/application/api/ApiService';
 import { APP_ROUTE } from '@/constants';
-import type { Topics } from '@/features/topics/types/topicService';
 export function topicsRequest(request: Request): ApiAdapters['getTopics'] {
-  return async () => {
+  return async (projectID: string) => {
     try {
-      const response = (await request(`${APP_ROUTE.TOPICS}`, {
+      const response = (await request(`${APP_ROUTE.PROJECTS}/${projectID}/topics`, {
         method: 'GET',
       })) as any;
 
-      return getValidApiResponse<Topics>(response);
+      return getValidApiResponse<any>(response);
```

Comment:
Any reason for this `Any` instead of `Topics` type ?

Reply:
I got an error here when I had `Topics` as the type. I've removed the `try/catch` block and now there isn't an error when `Topics` is listed as the type.